package.json

```diff
@@ -391,6 +391,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "Whether to automatically send a 'hot reload' request during a Flutter debug session when saving files."
+				},
+				"dart.analyzeAngularTemplates": {
```

Comment:
I've renamed the flag to previewAnalyzeAngularTemplates, removed it from package.json, added a v2.4.0 section to CHANGELOG.md and added a note about this flag there